### ISSUE TITLE
Enhancement, glyph mirror/flip ?

### ISSUE THREAD

**Issue by tablatronix** [olikraus/u8g2#2035]
Being able to flip a glyph might be useful, to get its H or V mirror. NOT rotation.

**Comment by olikraus** [comment]
Hmm I don't know any useful application for that

**Comment by tablatronix** [comment]
Icons, any UI directional symbols, animations etc

**Comment by olikraus** [comment]
Could be more an idea for the drawXBM procedure

**Comment by olikraus** [comment]
Let me have a look into this...

**Comment by tablatronix** [comment]
Yeah I thought it was be a simple matrix operation per char et.al. setFontDirection, But I didn't actually look, probably not worth the effort.
